composer.json

```diff
@@ -30,11 +31,14 @@
     "autoload-dev": {
         "psr-4" : {
             "Bugsnag\\BugsnagLaravel\\Tests\\" : "tests/"
-        }
+        },
+        "files": [
+            "tests/bc.php"
+        ]
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.7-dev"
+            "dev-master": "2.14-dev"
```

Comment:
Remember to update this branch alias to reflect the current dev state of master. :)

Reply:
Added this to the release instructions